notebooks/notebook_support.py

```diff
@@ -28,8 +43,143 @@
 
 MODEL_PATH = Path().resolve().parent / "data" / "models"
 
+logger = logging.getLogger(__name__)
 
-def new_resnet_model(output_size: int) -> TorchModel:
+
+class TorchLogisticRegression(nn.Module):
```

Comment:
Won't this fail if torch is not installed? I think all torch-dependent methods need to be factored out into their own module, which is only imported when torch is available.